app/src/main/java/com/keylesspalace/tusky/components/account/AccountActivity.kt

```diff
@@ -68,7 +68,9 @@ import com.keylesspalace.tusky.interfaces.LinkListener
 import com.keylesspalace.tusky.interfaces.ReselectableFragment
 import com.keylesspalace.tusky.settings.PrefKeys
 import com.keylesspalace.tusky.util.DefaultTextWatcher
+import com.keylesspalace.tusky.util.Error
```

Comment:
We had the wrong import for this all the time 😱 
This new inspection highlighted it as a warning, I would have never seen that otherwise